content/personal/telepathicgrunt/open-source-licenses.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Open Source Licenses: A Quick Guide"
+date: 2025-03-31T00:33:12+08:00
+categories:
+- Personal Blog
+author: telepathicgrunt
+summary: "There are numerous licenses and options one can pick from to use for their mod. We are going to dive into a small subset of them, Open-Source Licenses."
+description: "This guide will talk about what are Open-Source Licenses, why they are good for the community, go into detail about some commonly used Open-Source Licenses, and what to watch out for with some. It is not a fully comprehensive guide but is a good strt. Reminder, neither me nor the NeoForge team are lawyers and anything we say cannot be taken as legal advice."
+---
+
+{{< box note >}}
+Reminder, neither me nor the NeoForge team are lawyers and anything we say here cannot be taken as legal advice.
+{{< /box >}}
+
+The reason for this blog is the NeoForge team as a whole, feel that Open-Source license is a good practice to encourage in the modding community. It allows modders to learn from each other and be able to contribute code to repositories or fork a mod if that mod goes abandoned.
```

Comment:
Why is "Open-Source" hyphenated? Also, are we using the definition of Open Source as published by the Open Source Initiative (https://opensource.org/osd)?